src/components/views/room_settings/RoomProfileSettings.js

```diff
@@ -150,7 +159,11 @@ export default class RoomProfileSettings extends React.Component {
         const AccessibleButton = sdk.getComponent('elements.AccessibleButton');
         const AvatarSetting = sdk.getComponent('settings.AvatarSetting');
         return (
-            <form onSubmit={this._saveProfile} autoComplete="off" noValidate={true}>
+            <form
+                onSubmit={this._saveProfile}
+                autoComplete="off" noValidate={true}
```

Comment:
one per line

Reply:
This isn't a rule we have?

Reply:
Neither is the indent-by-4 vs natural indenting in WebStorm, its a convention

Reply:
We also have a history of grouping similar properties onto a single line.